ir-engine/etherealengine-archive
# IR-1293 click to select mesh objects in scene edit mode

## Summary
enabling click to select in scene, fixing interactable issue from studio update


## References
closes [https://tsu.atlassian.net/browse/IR-1293](https://tsu.atlassian.net/browse/IR-1293)

## QA Steps
while in edit mode, try single-clicking on an object with a mesh, it should become your selected entity in the hierarchy view
